.pipelines/containerSourceData/scripts/BuildGoldenDistrolessContainer.sh

```diff
@@ -22,6 +22,13 @@ function DockerBuild {
 
     # Create container
     echo "+++ Create container $containerName"
+
+    # DOCKER_BUILDKIT=0 is set to avoid the unknown timeout error in the Azure DevOps pipeline.
+    # The error is likely caused by some BuildKit feature in version 24.0.9 of moby-engine.
+    # The error is not seen in the local environment.
+    # Setting DOCKER_BUILDKIT=0 disables BuildKit and uses the legacy builder.
+    # TODO: Remove this line once the issue is resolved.
```

Comment:
Do we have a bug tracking this issue, so we won't lose it?

Reply:
Good call. I am creating one right now and will link it in this PR.